templates/broker/broker.conf.erb

```diff
@@ -38,7 +38,7 @@ DEFAULT_MAX_TRACKED_ADDTL_STORAGE_PER_GEAR="0"
 <% if scope.lookupvar('::openshift_origin::mongodb_replicasets') == true %>
 MONGO_HOST_PORT="<%= scope.lookupvar('::openshift_origin::mongodb_replicasets_members').join(',') %>"
 <% else %>
- MONGO_HOST_PORT="<%= scope.lookupvar('::openshift_origin::datastore_fqdn') %>:<%= scope.lookupvar('::openshift_origin::mongodb_port') %>"
+MONGO_HOST_PORT="<%= scope.lookupvar('::openshift_origin::datastore_fqdn') %>:<%= scope.lookupvar('::openshift_origin::mongodb_port') %>"
```

Comment:
I removed it because an ERB file does not use indentation on conditional statements.
Therefore the end result is not as good ( aesthetically speaking)...